### PR TITLE
Issue/2493

### LINKED ISSUE #2493: Error Log receives messages WordPress database error Duplicate column
## Issue Overview
This error is generated when user is not logged in and `email_access_installed` key is set to `false`. This is a rare case which can be generated. However, we will be adding an additional check to make sure that the notice doesn't reproduce.

I've tried upgrading from `1.8.16` to `1.8.17` and it has no notice generated in error log. However, manipulating the give settings from database can reproduce issue.

## Expected Behavior
No Notice should generate in error log.

## Current Behavior
`WordPress database error Duplicate column` notice generated in error log.

## Todos
- [ ] Tests
- [ ] Documentation

### PR DESCRIPTION
**Pull request by mehul0810**
## Description
This PR resolved #2493 

## How Has This Been Tested?
I've tested this by setting `email_access_installed` to `false` and then confirming whether the notice still appears in error log or not.

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/class-give-email-access.php [hunk 1]
```diff
@@ -412,12 +412,19 @@ public function create_columns() {
...
 
-		// Create columns in customers table
-		$query = $wpdb->query( "ALTER TABLE {$wpdb->prefix}give_customers ADD `token` VARCHAR(255) CHARACTER SET utf8 NOT NULL, ADD `verify_key` VARCHAR(255) CHARACTER SET utf8 NOT NULL AFTER `token`, ADD `verify_throttle` DATETIME NOT NULL AFTER `verify_key`" );
+		$is_token_exists           = Give()->donors->is_column_exists( 'token' );
+		$is_verify_key_exists      = Give()->donors->is_column_exists( 'verify_key' );
+		$is_verify_throttle_exists = Give()->donors->is_column_exists( 'verify_throttle' );
 
-		// Columns added properly
-		if ( $query ) {
-			give_update_option( 'email_access_installed', 1 );
```
Comment (ravinderk) [inline includes/class-give-email-access.php]: @mehul0810 Write automatic upgrade to remove `email_access_installed`.

Comment (mehul0810) [reply includes/class-give-email-access.php]: @ravinderk Added Automatic Upgrade as well as improved the functioning of creating columns as discussed over call.

File: includes/class-give-db.php [hunk 2]
```diff
@@ -335,6 +335,33 @@ public function table_exists( $table ) {
...
+	 *
+	 * @param string $column_name Name of the Column in Database Table.
+	 *
+	 * @since 1.8.18
+	 *
+	 * @see https://gist.github.com/datafeedr/54e89e07f87232fb055121bb766743fe
+	 *
+	 * @return bool
+	 */
+	public function is_column_exists( $column_name ) {
```
Comment (DevinWalker) [inline includes/class-give-db.php]: @mehul0810 I'd love to get this renamed to `does_column_exist( $column_name )` pre-1.8.18 release

Comment (DevinWalker) [reply includes/class-give-db.php]: @mehul0810 I'll update it

**Pull request merged**